source/dub/internal/utils.d

```diff
@@ -359,10 +359,25 @@ auto bitFieldNames(T)(T value) if(is(T==enum) && isIntegral!T)
 }
 
 
+bool isIdentChar(dchar ch)
+{
+	return ch >= 'A' && ch <= 'Z' || ch >= 'a' && ch <= 'z' || ch >= '0' && ch <= '9' || ch == '_';
```

Comment:
I'll fix these. But it should be noted that this code is just moved from another file, so I wouldn't make all shortcomings of the existing code part of this PR, as that would maybe take months to process.